styles.css

```diff
@@ -14,32 +14,20 @@ body {
 }
 
 main {
-  background-color: #e5e5e5;
+  background-color: hsl(0, 0%, 90%);
   width: 100%;
 }
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible. You may consider fixing the introduction section of your page
- [ ] Also make sure your page does not overflow on the mobile version. we do not need to scroll to the whole page content. Kindly fix that.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201398331-a449c85a-409d-4b2b-a28b-90e0a7a35800.png)

![image](https://user-images.githubusercontent.com/97100091/201398661-29dd6991-05ba-47b4-9b83-1b3bb4ce7d22.png)
![image](https://user-images.githubusercontent.com/97100091/201398729-5464b400-4b49-4675-b2e5-e587e4d6dca0.png)


Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201398411-288a008a-d7d5-4e25-b518-75321b85c869.png)